src/views/Home/components/Schedule/schedule.scss

```diff
@@ -119,21 +116,10 @@
     right: 0;
   }
 
-  &__dayText {
-    @include typography.description;
-
-    position: absolute;
-    padding: var(--extra-small-spacer);
-    bottom: 40%;
-    margin-right: 20px;
-    left: calc(-2.3vw + 10px);
-    color: var(--pink);
-    transform: rotate(-90deg);
-  }
-
   &__title {
     @include typography.alt-title-text;
 
+    color: var(--salmon);
```

Comment:
Instead of adding `--salmon` here, can we change `--pink` to `--salmon` in the corresponding typography file?

Reply:
Yeah I can do that! If I were to change salmon to pink would we have to change all the references to salmon?

Reply:
I think it should be fine if both are defined in `variables` and correlate to the same hex code. I think @ilopezro mentioned cleaning up the code later!